virt/virt-4-8-release-notes.adoc

```diff
@@ -63,12 +63,12 @@ The SVVP Certification applies to:
 * You can use the Kubernetes NMSstate Operator to xref:../virt/node_network/virt-updating-node-network-config.adoc#virt-example-nmstate-IP-management_virt-updating-node-network-config[configure and manage IP addresses] on your cluster nodes.
 
 //CNV-11390 OpenShift Virtualization now supports live-migration of VM connected to an SR-IOV network.
-* {VirtProductName} now supports xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[live migration of virtual machines] that are attached to an SR-IOV network interface if the `sriovLiveMigration` feature gate is enabled in the `HyperConverged` custom resource (CR). 
+* {VirtProductName} now supports xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[live migration of virtual machines] that are attached to an SR-IOV network interface if the `sriovLiveMigration` feature gate is enabled in the `HyperConverged` custom resource (CR).
 
 [id="virt-4-8-storage-new"]
 === Storage
 
-//CNV-12268 Cloning a DataVolume into a different namespace (ie. when creating a VM from a template) is now faster and more efficient when using storage that supports CSI Snapshots.
+* Cloning a data volume into a different namespace is now faster and more efficient when using storage that supports Container Storage Interface (CSI) Snapshots. CDI automatically uses CSI Snapshots when they are available to improve performance when you create a virtual machine from a template.
```

Comment:
s/Snapshots/snapshots

Also, you might want to spell out CDI and add commas - "The Containerized Data Importer uses CSI snapshots, when they are available, to improve performance when you create a virtual machine from a template."

Reply:
Done.